Blog post about the Mitiq workshop (#499)
unitaryfund/unitary.fund: src/content/blog/2024_Mitiq_Workshop_QNumerics.md

The program's emphasis on practical, experiential learning with an emphasis on programming skills and hackathon-style sessions each evening made it a natural fit for hosting the Mitiq workshop.
Other topics covered in the summer school include general software engineering practices and cluster computing tools, advanced general scientific programming, GPU programming, fast general purpose wavefunction simulation, tensor networks for faster approximate quantum simulations, stabilizer formalism for quantum error correction, discrete event simulations, quantum chemistry, symbolic computer algebra basics, optimal control of quantum hardware, APIs for control of commercial quantum hardware.

We look forward to presenting topics including quantum error mitigation (QEM) core concepts and techniques, Mitiq structure and interface, and deep dive into the techniques of Zero Noise Extrapolation and Digital Dynamical Decoupling.
In the later part of the session, we will explore QEM on simulated noisy backends with benchmarks and calibration.

> The workshop will conclude in the evening with a Mitiq hackathon.
> To learn more about [Mitiq](https://unitary.fund/research/mitiq/), an open source quantum error mitigation compiler with over 100k downloads and developed by the Unitary Fund technical team with a community of over 50 contributors worldwide, see the [Mitiq documentation](https://mitiq.readthedocs.io/en/stable/).

Reviewer: ```suggestion
The workshop will conclude in the evening with a Mitiq hackathon.

The Qnumerics Summer school lecturers include Stefan Krastanov (U. Mass. Amherst), Katharine Hyatt (AWS Quantum), Roger Luo (QuEra), Miles Stoudenmire (Flatiron Institute) and Unitary Fund's Nathan Shammah and Misty Wahl. Prof. Stefan Krastanov, one of the organizers commented: 
> At JuliaCon 2023 Katharine and I were discussing how much of the opensource tooling created in the field is still underutilized and thought why not do something about it. The NSF Center for Quantum Networks (and now also the Unitary Fund) seemed to have agreed with us and were eager to sponsor a hacker-oriented summer school rife with late-night hackathons. We are all particularly excited that what we teach the students will be a force multiplier for their domain-specific skills, leading to rapid research progress into the many low-hanging fruits still present in the field of quantum information science. We are hopeful that quite a few research papers will start at our hackathons. 

To learn more about [Mitiq](https://unitary.fund/research/mitiq/), an open source quantum error mitigation compiler with over 100k downloads and developed by the Unitary Fund technical team with a community of over 50 contributors worldwide, see the [Mitiq documentation](https://mitiq.readthedocs.io/en/stable/).
```

The program's focus on practical, experiential learning with an emphasis on programming skills and hackathon-style sessions each evening made it a natural fit for hosting the Mitiq workshop.
Other topics covered in the summer school include general software engineering practices and cluster computing tools, advanced general scientific programming, GPU programming, fast general purpose wavefunction simulation, tensor networks for faster approximate quantum simulations, stabilizer formalism for quantum error correction, discrete event simulations, quantum chemistry, symbolic computer algebra basics, optimal control of quantum hardware, APIs for control of commercial quantum hardware.

We look forward to presenting topics including quantum error mitigation (QEM) core concepts and techniques, Mitiq structure and interface, and deep dive into the techniques of Zero Noise Extrapolation and Digital Dynamical Decoupling.
In the later part of the session, we will explore QEM on simulated noisy backends with benchmarks and calibration.

> The workshop will conclude in the evening with a Mitiq hackathon.
> To learn more about [Mitiq](https://unitary.fund/research/mitiq/), an open source quantum error mitigation compiler with over 100k downloads and developed by the Unitary Fund technical team with a community of over 70 contributors worldwide, see the [Mitiq documentation](https://mitiq.readthedocs.io/en/stable/).

Reviewer: ```suggestion
The workshop will conclude in the evening with a Mitiq hackathon.
The Qnumerics Summer school lecturers include Stefan Krastanov (U. Mass. Amherst), Katharine Hyatt (AWS Quantum), Roger Luo (QuEra), Miles Stoudenmire (Flatiron Institute) and Unitary Fund's Nathan Shammah and Misty Wahl. Prof. Stefan Krastanov, one of the organizers commented: 
> At JuliaCon 2023 Katharine and I were discussing how much of the opensource tooling created in the field is still underutilized and thought why not do something about it. The NSF Center for Quantum Networks (and now also the Unitary Fund) seemed to have agreed with us and were eager to sponsor a hacker-oriented summer school rife with late-night hackathons. We are all particularly excited that what we teach the students will be a force multiplier for their domain-specific skills, leading to rapid research progress into the many low-hanging fruits still present in the field of quantum information science. We are hopeful that quite a few research papers will start at our hackathons. 
To learn more about [Mitiq](https://unitary.fund/research/mitiq/), an open source quantum error mitigation compiler with over 100k downloads and developed by the Unitary Fund technical team with a community of over 70 contributors worldwide, see the [Mitiq documentation](https://mitiq.readthedocs.io/en/stable/). Mitiq's open source ecosystem growth is accelerated by the National Science Foundation (NSF) POSE program (Phase II) [awarded](https://unitary.fund/posts/2023_mitiq_nsf_pose/) to Unitary Fund.
```

---

> title: Announcing the Mitiq Workshop at the Numerical Methods in Quantum Information Science Summer School
> author: Misty Wahl, Nathan Shammah
> day: 3

Reviewer: ```suggestion
title: Announcing the Mitiq Workshop at the QNumerics Summer School
author: Misty Wahl and Nathan Shammah
day: 5
```

month: 7
year: 2024
tags: 
  - mitiq
  - workshop
  - nsf
---

> We are pleased to announce the first in-person Mitiq workshop, which will be held as part of the [Numerical Methods in Quantum Information Science (QIS) Summer School](https://qnumerics.org/) at the Mount Ida campus of University of Massachusetts Amherst. The School spans through August 12-18, 2024, with Saturday, August 17, 2024 fully dedicated to Mitiq and quantum error mitigation.

Reviewer: ```suggestion
**We are pleased to announce the first in-person workshop for the Mitiq project.**

![](/images/qnumerics_logo.png)

## About the QNumerics Summer School
The workshop will be held as part of the [Numerical Methods in Quantum Information Science (QIS) Summer School](https://qnumerics.org/) at the Mount Ida campus of University of Massachusetts Amherst.** 

The School spans through August 12-18, 2024, with Saturday, August 17, 2024 fully dedicated to Mitiq and quantum error mitigation.

```

The program's focus on practical, experiential learning with an emphasis on programming skills and hackathon-style sessions each evening made it a natural fit for hosting the Mitiq workshop.
Other topics covered in the Summer school include general software engineering practices and cluster computing tools, advanced general scientific programming, GPU programming, fast general purpose wavefunction simulation, tensor networks for faster approximate quantum simulations, stabilizer formalism for quantum error correction, discrete event simulations, quantum chemistry, symbolic computer algebra basics, optimal control of quantum hardware, APIs for control of commercial quantum hardware. 

> A focus of the QNumerics School is placed on [Julia](https://julialang.org/) as a transparent and efficient programming language designed for science. At Unitary Fund we're proud to support [several Julia projects](https://unitary.fund/grants/) (and sponsor this [JuliaCon 2024](https://juliacon.org/2024/)). 

Reviewer: ```suggestion
## A focus on open-source scientific tools for quantum programming
A focus of the QNumerics School is placed on [Julia](https://julialang.org/) as a transparent and efficient programming language designed for science. At Unitary Fund we're proud to support [several Julia projects](https://unitary.fund/grants/) (and sponsor this [JuliaCon 2024](https://juliacon.org/2024/)). 
```

We look forward to presenting topics including quantum error mitigation (QEM) core concepts and techniques, Mitiq structure and interface, and deep dive into the techniques of Zero Noise Extrapolation and Digital Dynamical Decoupling.
In the later part of the session, we will explore QEM on simulated noisy backends with benchmarks and calibration.
The workshop will conclude in the evening with a Mitiq hackathon.

> The Qnumerics Summer school lecturers include Stefan Krastanov (U. Mass. Amherst), Katharine Hyatt (AWS Quantum), Roger Luo (QuEra), Miles Stoudenmire (Flatiron Institute) and Unitary Fund's Nathan Shammah and Misty Wahl. Prof. Stefan Krastanov, one of the organizers commented: 

Reviewer: ```suggestion
The Summer School is supported by a number of organizations.  
![](/images/qnumerics_supporters.png)

The Qnumerics Summer school lecturers include Stefan Krastanov (U. Mass. Amherst), Katharine Hyatt (AWS Quantum), Roger Luo (QuEra), Miles Stoudenmire (Flatiron Institute) and Unitary Fund's Nathan Shammah and Misty Wahl. Prof. Stefan Krastanov, one of the organizers commented: 
```
